test/routes/groups.test.js

```diff
@@ -0,0 +1,31 @@
+import "dotenv/config";
+import express from "express";
+import mongoose from "mongoose";
+import request from "supertest";
+import router from "../../src/routes/groups.js";
+
+const app = express();
+
+app.use("/groups", router);
+
+// before all the route tests, connect to the database
+beforeAll(async () => {
+  await mongoose.connect(process.env.DB_CONNECTION, {
```

Comment:
we're setting up a test database, which will relaty on `TEST_DB_CONNECTION` ENV variable. 

Hold until test database established for project

Reply:
Sounds good 👍